### ISSUE TITLE
playing a HLG file flashes briefly at startup

### ISSUE THREAD

**Issue by kodabb** [mpv-player/mpv#4628]
### mpv version and platform
git master 9e124933e55f86992b1079aa6c1985bbe6721a28

### Reproduction steps
* Playback a HLG file. I tried with the `opengl-hq` backend and the default backend
* Initially I had thought it was due to seeking or the fact that it was paused but it is possible to trigger this without any command line option
* This does not happen if the file is HDR10

### Expected behavior
* The file should play normally

### Actual behavior
* The file flash for a frame or two, that is there is a spike in the frame brightness

### Log file
```
[cplayer] Command line options: '/Users/koda/Downloads/LG Cymatic Jazz 4K Demo.mp4' '-v' '--pause'
[cplayer] mpv 0.26.0-1-g9e124933e5 (C) 2000-2017 mpv/MPlayer/mplayer2 projects
[cplayer]  built on Wed Jul 19 13:49:13 CEST 2017
[cplayer] ffmpeg library versions:
[cplayer]    libavutil       55.67.100
[cplayer]    libavcodec      57.100.103
[cplayer]    libavformat     57.75.100
[cplayer]    libswscale      4.7.101
[cplayer]    libavfilter     6.94.100
[cplayer]    libswresample   2.8.100
[cplayer] ffmpeg version: N-86724-g9086cbdc90
[cplayer] 
[cplayer] Configuration: ./waf configure
[cplayer] List of enabled features: 51deb apple-remote asm atomics avutil-content-light-level avutil-imgcpy-uc bsd-fstatfs build-date cocoa coreaudio cplayer cplugins cuda-hwaccel debug-build encoding fchmod gl gl-cocoa gl-x11 gnuc iconv is_ffmpeg jack lcms2 libass libass-osd libav libavcodec libdl libm lua macos-touchbar nanosleep optimize osx-thread-name posix posix-or-mingw posix-spawn pthreads pulse shm stdatomic termios videotoolbox-gl videotoolbox-hwaccel videotoolbox-hwaccel-new x11 xv zlib
[global] config path: '' -> '/Users/koda/.config/mpv'
[global] config path: 'encoding-profiles.conf' -/-> '/Users/koda/.config/mpv/encoding-profiles.conf'
[global] config path: 'encoding-profiles.conf' -/-> '/Users/koda/.mpv/encoding-profiles.conf'
[global] config path: 'encoding-profiles.conf' -/-> '/Users/koda/src/mpv/build/encoding-profiles.conf'
[global] config path: 'encoding-profiles.conf' -> '/usr/local/etc/mpv/encoding-profiles.conf'
[cplayer] Reading config file /usr/local/etc/mpv/encoding-profiles.conf
[cplayer] Option fmt: unknown format name: '444p'
[cplayer] Error parsing option fmt (option parameter could not be parsed)
[cplayer] Option vf-add: Error while parsing noformat parameter fmt (444p)
[cplayer] Error parsing option vf-add (option parameter could not be parsed)
[cplayer] /usr/local/etc/mpv/encoding-profiles.conf:78: setting option vf-add='noformat=444p,noformat=444p9,noformat=444p10,noformat=422p,noformat=422p9,noformat=422p10' failed.
[cplayer] Passing multiple arguments to ofopts-add is deprecated!
[cplayer] Passing multiple arguments to ovcopts-add is deprecated!
[cplayer] Passing multiple arguments to ofopts-add is deprecated!
[cplayer] Passing multiple arguments to ovcopts-add is deprecated!
[cplayer] Passing multiple arguments to ovcopts-add is deprecated!
[global] config path: 'mpv.conf' -/-> '/Users/koda/.config/mpv/mpv.conf'
[global] config path: 'config' -/-> '/Users/koda/.config/mpv/config'
[global] config path: 'mpv.conf' -/-> '/Users/koda/.mpv/mpv.conf'
[global] config path: 'config' -/-> '/Users/koda/.mpv/config'
[global] config path: 'mpv.conf' -/-> '/Users/koda/src/mpv/build/mpv.conf'
[global] config path: 'config' -/-> '/Users/koda/src/mpv/build/config'
[global] config path: 'mpv.conf' -/-> '/usr/local/etc/mpv/mpv.conf'
[global] config path: 'config' -/-> '/usr/local/etc/mpv/config'
[cplayer] Setting option 'v' = '' (flags = 8)
[cplayer] Setting option 'pause' = '' (flags = 8)
[global] config path: 'input.conf' -/-> '/Users/koda/.config/mpv/input.conf'
[global] config path: 'input.conf' -/-> '/Users/koda/.mpv/input.conf'
[global] config path: 'input.conf' -/-> '/Users/koda/src/mpv/build/input.conf'
[global] config path: 'input.conf' -/-> '/usr/local/etc/mpv/input.conf'
[osc] Loading lua script @osc.lua...
[global] config path: 'scripts' -/-> '/Users/koda/.config/mpv/scripts'
[global] config path: 'scripts' -/-> '/Users/koda/.mpv/scripts'
[global] config path: 'scripts' -/-> '/Users/koda/src/mpv/build/scripts'
[global] config path: 'scripts' -/-> '/usr/local/etc/mpv/scripts'
[osc] loading mp.defaults
[osc] loading @osc.lua
[global] config path: 'lua-settings/osc.conf' -/-> '/Users/koda/.config/mpv/lua-settings/osc.conf'
[global] config path: 'lua-settings/osc.conf' -/-> '/Users/koda/.mpv/lua-settings/osc.conf'
[global] config path: 'lua-settings/osc.conf' -/-> '/Users/koda/src/mpv/build/lua-settings/osc.conf'
[global] config path: 'lua-settings/osc.conf' -/-> '/usr/local/etc/mpv/lua-settings/osc.conf'
[osc] lua-settings/osc.conf not found. 
[cplayer] Run command: define-section, flags=0, args=[showhide, mouse_move script-binding osc/__keybinding1
[cplayer] mouse_leave script-binding osc/__keybinding2
[cplayer] , force]
[cplayer] Run command: enable-section, flags=0, args=[showhide, allow-hide-cursor+allow-vo-dragging]
[cplayer] Run command: define-section, flags=0, args=[input, mouse_btn0 script-binding osc/__keybinding3
[cplayer] shift+mouse_btn0 script-binding osc/__keybinding4
[cplayer] mouse_btn2 script-binding osc/__keybinding5
[cplayer] mouse_btn3 script-binding osc/__keybinding6
[cplayer] mouse_btn4 script-binding osc/__keybinding7
[cplayer] axis_up script-binding osc/__keybinding8
[cplayer] axis_down script-binding osc/__keybinding9
[cplayer] mouse_btn0_dbl ignore
[cplayer] shift+mouse_btn0_dbl ignore
[cplayer] mouse_btn2_dbl ignore
[cplayer] , force]
[cplayer] Run command: enable-section, flags=0, args=[input, ]
[cplayer] Run command: define-section, flags=0, args=[input_osc, del script-binding osc/__keybinding10
[cplayer] , default]
[cplayer] Run command: enable-section, flags=0, args=[input_osc, allow-hide-cursor+allow-vo-dragging]
[cplayer] Run command: define-section, flags=0, args=[input_forced_osc, , force]
[cplayer] Run command: enable-section, flags=0, args=[input_forced_osc, allow-hide-cursor+allow-vo-dragging]
[cplayer] Done loading @osc.lua.
[ytdl_hook] Loading lua script @ytdl_hook.lua...
[global] config path: 'scripts' -/-> '/Users/koda/.config/mpv/scripts'
[global] config path: 'scripts' -/-> '/Users/koda/.mpv/scripts'
[global] config path: 'scripts' -/-> '/Users/koda/src/mpv/build/scripts'
[global] config path: 'scripts' -/-> '/usr/local/etc/mpv/scripts'
[ytdl_hook] loading mp.defaults
[cplayer] Run command: disable-section, flags=0, args=[input]
[global] config path: 'fonts' -/-> '/Users/koda/.config/mpv/fonts'
[global] config path: 'fonts' -/-> '/Users/koda/.mpv/fonts'
[global] config path: 'fonts' -/-> '/Users/koda/src/mpv/build/fonts'
[global] config path: 'fonts' -/-> '/usr/local/etc/mpv/fonts'
[osd/libass] Shaper: FriBidi 0.19.7 (SIMPLE) HarfBuzz-ng 1.4.6 (COMPLEX)
[global] config path: 'subfont.ttf' -/-> '/Users/koda/.config/mpv/subfont.ttf'
[global] config path: 'subfont.ttf' -/-> '/Users/koda/.mpv/subfont.ttf'
[global] config path: 'subfont.ttf' -/-> '/Users/koda/src/mpv/build/subfont.ttf'
[global] config path: 'subfont.ttf' -/-> '/usr/local/etc/mpv/subfont.ttf'
[global] config path: 'fonts.conf' -/-> '/Users/koda/.config/mpv/fonts.conf'
[global] config path: 'fonts.conf' -/-> '/Users/koda/.mpv/fonts.conf'
[global] config path: 'fonts.conf' -/-> '/Users/koda/src/mpv/build/fonts.conf'
[global] config path: 'fonts.conf' -/-> '/usr/local/etc/mpv/fonts.conf'
[osd/libass] Setting up fonts...
[ytdl_hook] loading @ytdl_hook.lua
[osd/libass] Using font provider coretext
[osd/libass] Done.
[global] config path: 'lua-settings/ytdl_hook.conf' -/-> '/Users/koda/.config/mpv/lua-settings/ytdl_hook.conf'
[global] config path: 'lua-settings/ytdl_hook.conf' -/-> '/Users/koda/.mpv/lua-settings/ytdl_hook.conf'
[global] config path: 'lua-settings/ytdl_hook.conf' -/-> '/Users/koda/src/mpv/build/lua-settings/ytdl_hook.conf'
[global] config path: 'lua-settings/ytdl_hook.conf' -/-> '/usr/local/etc/mpv/lua-settings/ytdl_hook.conf'
[ytdl_hook] lua-settings/ytdl_hook.conf not found. 
[cplayer] Run command: hook-add, flags=0, args=[on_load, 1, 10]
[cplayer] Run command: hook-add, flags=0, args=[on_preloaded, 2, 10]
[cplayer] Done loading @ytdl_hook.lua.
[global] config path: 'scripts' -/-> '/Users/koda/.config/mpv/scripts'
[global] config path: 'scripts' -/-> '/Users/koda/.mpv/scripts'
[global] config path: 'scripts' -/-> '/Users/koda/src/mpv/build/scripts'
[global] config path: 'scripts' -/-> '/usr/local/etc/mpv/scripts'
[global] config path: 'watch_later' -> '/Users/koda/.config/mpv/watch_later'
[cplayer] Playing: /Users/koda/Downloads/LG Cymatic Jazz 4K Demo.mp4
[cplayer] Running hook: ytdl_hook/on_load
[cplayer] Run command: hook-ack, flags=0, args=[on_load]
[file] Opening /Users/koda/Downloads/LG Cymatic Jazz 4K Demo.mp4
[file] Stream opened successfully.
[demux] Trying demuxers for level=normal.
[lavf] Found 'mov,mp4,m4a,3gp,3g2,mj2' at score=100 size=2048.
[demux] Detected file format: mov,mp4,m4a,3gp,3g2,mj2 (libavformat)
[cplayer] Opening done: /Users/koda/Downloads/LG Cymatic Jazz 4K Demo.mp4
[find_files] Loading external files in /Users/koda/Downloads/
[global] config path: 'sub' -/-> '/Users/koda/.config/mpv/sub'
[global] config path: 'sub' -/-> '/Users/koda/.mpv/sub'
[global] config path: 'sub' -> '/Users/koda/src/mpv/build/sub'
[find_files] Loading external files in /Users/koda/src/mpv/build/sub
[cplayer] Running hook: ytdl_hook/on_preloaded
[cplayer] Run command: hook-ack, flags=0, args=[on_preloaded]
[cplayer]  (+) Video --vid=1 (*) (hevc 3840x2160 59.943fps)
[vo/opengl] Initializing OpenGL backend 'cocoa'
[vo/opengl] GL_VERSION='4.1 INTEL-10.25.13'
[vo/opengl] Detected desktop OpenGL 4.1.
[vo/opengl] GL_VENDOR='Intel Inc.'
[vo/opengl] GL_RENDERER='Intel Iris Pro OpenGL Engine'
[vo/opengl] GL_SHADING_LANGUAGE_VERSION='4.10'
[vo/opengl] Loaded extension GL_APPLE_rgb_422.
[vo/opengl] 16 bit texture depth: 16.
[vo/opengl] Testing FBO format 0x805b
[vo/opengl] Create FBO: 16x16 (16x16)
[vo/opengl] Using FBO format 0x805b.
[vo/opengl] No advanced processing required. Enabling dumb mode.
[vo/opengl] Assuming 59.990270 FPS for display sync.
[vd] Container reported FPS: 59.943012
[vd] Codec list:
[vd]     hevc - HEVC (High Efficiency Video Coding)
[vd] Opening video decoder hevc
[vd] Using software decoding.
[vd] Detected 8 logical cores.
[vd] Requesting 9 threads for decoding.
[vd] Selected video codec: hevc (HEVC (High Efficiency Video Coding))
[cplayer] Starting playback...
[vd] Decoder format: 3840x2160 yuv420p10 bt.2020-ncl/bt.2020/bt.1886/limited CL=unknown
[vd] Using container aspect ratio.
[vf] Video filter chain:
[vf]   [in] 3840x2160 yuv420p10 bt.2020-ncl/bt.2020/bt.1886/limited SP=1.000000 CL=unknown
[vf]   [out] 3840x2160 yuv420p10 bt.2020-ncl/bt.2020/bt.1886/limited SP=1.000000 CL=unknown
[cplayer] VO: [opengl] 3840x2160 yuv420p10
[cplayer] VO: Description: Extended OpenGL Renderer
[vo/opengl] Resize: 7680x4320
[vo/opengl] Window size: 7680x4320
[vo/opengl] Video source: 3840x2160 (1:1)
[vo/opengl] Video display: (0, 0) 3840x2160 -> (0, 0) 7680x4320
[vo/opengl] Video scale: 2.000000/2.000000
[vo/opengl] OSD borders: l=0 t=0 r=0 b=0
[vo/opengl] Video borders: l=0 t=0 r=0 b=0
[vo/opengl] Testing FBO format 0x805b
[vo/opengl] Create FBO: 16x16 (16x16)
[vo/opengl] Using FBO format 0x805b.
[vo/opengl] No advanced processing required. Enabling dumb mode.
[vo/opengl] Texture for plane 0: 3840x2160
[vo/opengl] Texture for plane 1: 1920x1080
[vo/opengl] Texture for plane 2: 1920x1080
[vo/opengl] Resize: 7680x4320
[vo/opengl] Window size: 7680x4320
[cplayer] set video colors output-levels=0 
[vo/opengl] Video source: 3840x2160 (1:1)
[vo/opengl] Video display: (0, 0) 3840x2160 -> (0, 0) 7680x4320
[vo/opengl] Video scale: 2.000000/2.000000
[vo/opengl] OSD borders: l=0 t=0 r=0 b=0
[vo/opengl] Video borders: l=0 t=0 r=0 b=0
[vo/opengl] new shader program:
[vo/opengl] [  1] color.r = 64.250000 * vec4(texture(texture0, texcoord0)).r;
[vo/opengl] [  2] color.g = 64.250000 * vec4(texture(texture1, texcoord1)).r;
[vo/opengl] [  3] color.b = 64.250000 * vec4(texture(texture2, texcoord2)).r;
[vo/opengl] [  4] color = color.rgbr;
[vo/opengl] [  5] color.rgb = mat3(colormatrix) * color.rgb + colormatrix_c;
[vo/opengl] [  6] color.a = 1.0;
[vo/opengl] [  7] // color mapping
[vo/opengl] [  8] // linearize
[vo/opengl] [  9] color.rgb = clamp(color.rgb, 0.0, 1.0);
[vo/opengl] [ 10] color.rgb = pow(color.rgb, vec3(2.4));
[vo/opengl] [ 11] color.rgb *= vec3(1.0/1.000000);
[vo/opengl] [ 12] color.rgb = cms_matrix * color.rgb;
[vo/opengl] [ 13] // delinearize
[vo/opengl] [ 14] color.rgb = clamp(color.rgb, 0.0, 1.0);
[vo/opengl] [ 15] color.rgb *= vec3(1.000000);
[vo/opengl] [ 16] color.rgb = pow(color.rgb, vec3(1.0/2.4));
[vo/opengl] shader link log (status=1): WARNING: Output of vertex shader 'texcoord5' not read by fragment shader
[vo/opengl] WARNING: Output of vertex shader 'texcoord3' not read by fragment shader
[vo/opengl] WARNING: Output of vertex shader 'texcoord4' not read by fragment shader
[vo/opengl] 
[osd/libass] fontselect: (sans-serif, 400, 0) -> /System/Library/Fonts/Helvetica.dfont, -1, Helvetica
[cplayer] first video frame after restart shown
[cplayer] playback restart complete
[statusline] (Paused) V: 00:00:00 / 00:04:57 (0%)
[cplayer] Run command: cycle, flags=9, args=[pause, (NULL)]
[cplayer] Set property: pause -> 1
[vd] Decoder format: 3840x2160 yuv420p10 bt.2020-ncl/bt.2020/hlg/limited CL=unknown
[vd] Using container aspect ratio.
[vf] Video filter chain:
[vf]   [in] 3840x2160 yuv420p10 bt.2020-ncl/bt.2020/hlg/limited SP=10.000000 CL=unknown
[vf]   [out] 3840x2160 yuv420p10 bt.2020-ncl/bt.2020/hlg/limited SP=10.000000 CL=unknown
[statusline] V: 00:00:00 / 00:04:57 (0%)
[cplayer] VO: [opengl] 3840x2160 yuv420p10
[cplayer] VO: Description: Extended OpenGL Renderer
[vo/opengl] Resize: 7680x4320
[vo/opengl] Window size: 7680x4320
[vo/opengl] Video source: 3840x2160 (1:1)
[vo/opengl] Video display: (0, 0) 3840x2160 -> (0, 0) 7680x4320
[vo/opengl] Video scale: 2.000000/2.000000
[vo/opengl] OSD borders: l=0 t=0 r=0 b=0
[vo/opengl] Video borders: l=0 t=0 r=0 b=0
[vo/opengl] Testing FBO format 0x805b
[vo/opengl] Create FBO: 16x16 (16x16)
[vo/opengl] Using FBO format 0x805b.
[vo/opengl] No advanced processing required. Enabling dumb mode.
[vo/opengl] Texture for plane 0: 3840x2160
[vo/opengl] Texture for plane 1: 1920x1080
[vo/opengl] Texture for plane 2: 1920x1080
[cplayer] set video colors output-levels=0 
[statusline] V: 00:00:00 / 00:04:57 (0%)
[vo/opengl] new shader program:
[vo/opengl] [  1] color.r = 64.250000 * vec4(texture(texture0, texcoord0)).r;
[vo/opengl] [  2] color.g = 64.250000 * vec4(texture(texture1, texcoord1)).r;
[vo/opengl] [  3] color.b = 64.250000 * vec4(texture(texture2, texcoord2)).r;
[vo/opengl] [  4] color = color.rgbr;
[vo/opengl] [  5] color.rgb = mat3(colormatrix) * color.rgb + colormatrix_c;
[vo/opengl] [  6] color.a = 1.0;
[vo/opengl] [  7] // color mapping
[vo/opengl] [  8] // linearize
[vo/opengl] [  9] color.rgb = clamp(color.rgb, 0.0, 1.0);
[vo/opengl] [ 10] color.rgb = mix(vec3(4.0) * color.rgb * color.rgb,
[vo/opengl] [ 11]                 exp((color.rgb - vec3(0.559911)) * vec3(1.0/0.178833)) + vec3(0.284669),
[vo/opengl] [ 12]                 lessThan(vec3(0.5), color.rgb));
[vo/opengl] [ 13] color.rgb *= vec3(1.0/12.000000);
[vo/opengl] [ 14] // apply ootf
[vo/opengl] [ 15] color.rgb *= vec3(12.000000);
[vo/opengl] [ 16] color.rgb *= vec3(0.506970 * pow(dot(src_luma, color.rgb), 0.2));
[vo/opengl] [ 17] color.rgb *= vec3(1.0/12.000000);
[vo/opengl] [ 18] // rescale value range;
[vo/opengl] [ 19] color.rgb *= vec3(12.000000);
[vo/opengl] [ 20] // HDR tone mapping
[vo/opengl] [ 21] float luma = dot(src_luma, color.rgb);
[vo/opengl] [ 22] float luma_orig = luma;
[vo/opengl] [ 23] float overbright = max(luma - 2.000000, 1e-6) / max(luma, 1e-6);
[vo/opengl] [ 24] color.rgb = mix(color.rgb, vec3(luma), overbright);
[vo/opengl] [ 25] luma = mix(1.054444 * (luma + -0.085353) / (luma + 0.454444), luma, luma <= 0.300000);
[vo/opengl] [ 26] color.rgb *= luma / luma_orig;
[vo/opengl] [ 27] color.rgb = cms_matrix * color.rgb;
[vo/opengl] [ 28] // delinearize
[vo/opengl] [ 29] color.rgb = clamp(color.rgb, 0.0, 1.0);
[vo/opengl] [ 30] color.rgb *= vec3(1.000000);
[vo/opengl] [ 31] color.rgb = pow(color.rgb, vec3(1.0/2.2));
[vo/opengl] shader link log (status=1): WARNING: Output of vertex shader 'texcoord5' not read by fragment shader
[vo/opengl] WARNING: Output of vertex shader 'texcoord3' not read by fragment shader
[vo/opengl] WARNING: Output of vertex shader 'texcoord4' not read by fragment shader
[vo/opengl] 
[statusline] V: 00:00:00 / 00:04:57 (0%) Dropped: 2
[vd] Decoder format: 3840x2160 yuv420p10 bt.2020-ncl/bt.2020/bt.1886/limited CL=unknown
[vd] Using container aspect ratio.
[vf] Video filter chain:
[vf]   [in] 3840x2160 yuv420p10 bt.2020-ncl/bt.2020/bt.1886/limited SP=1.000000 CL=unknown
[vf]   [out] 3840x2160 yuv420p10 bt.2020-ncl/bt.2020/bt.1886/limited SP=1.000000 CL=unknown
[statusline] V: 00:00:00 / 00:04:57 (0%) Dropped: 3
[vd] Decoder format: 3840x2160 yuv420p10 bt.2020-ncl/bt.2020/hlg/limited CL=unknown
[vd] Using container aspect ratio.
[vf] Video filter chain:
[vf]   [in] 3840x2160 yuv420p10 bt.2020-ncl/bt.2020/hlg/limited SP=10.000000 CL=unknown
[vf]   [out] 3840x2160 yuv420p10 bt.2020-ncl/bt.2020/hlg/limited SP=10.000000 CL=unknown
[cplayer] VO: [opengl] 3840x2160 yuv420p10
[cplayer] VO: Description: Extended OpenGL Renderer
[vo/opengl] Resize: 7680x4320
[vo/opengl] Window size: 7680x4320
[vo/opengl] Video source: 3840x2160 (1:1)
[vo/opengl] Video display: (0, 0) 3840x2160 -> (0, 0) 7680x4320
[vo/opengl] Video scale: 2.000000/2.000000
[vo/opengl] OSD borders: l=0 t=0 r=0 b=0
[vo/opengl] Video borders: l=0 t=0 r=0 b=0
[vo/opengl] Testing FBO format 0x805b
[vo/opengl] Create FBO: 16x16 (16x16)
[vo/opengl] Using FBO format 0x805b.
[vo/opengl] No advanced processing required. Enabling dumb mode.
[vo/opengl] Texture for plane 0: 3840x2160
[vo/opengl] Texture for plane 1: 1920x1080
[vo/opengl] Texture for plane 2: 1920x1080
[cplayer] set video colors output-levels=0 
[statusline] V: 00:00:00 / 00:04:57 (0%)
[cplayer] VO: [opengl] 3840x2160 yuv420p10
[cplayer] VO: Description: Extended OpenGL Renderer
[vo/opengl] Resize: 7680x4320
[vo/opengl] Window size: 7680x4320
[vo/opengl] Video source: 3840x2160 (1:1)
[vo/opengl] Video display: (0, 0) 3840x2160 -> (0, 0) 7680x4320
[vo/opengl] Video scale: 2.000000/2.000000
[vo/opengl] OSD borders: l=0 t=0 r=0 b=0
[vo/opengl] Video borders: l=0 t=0 r=0 b=0
[vo/opengl] Testing FBO format 0x805b
[vo/opengl] Create FBO: 16x16 (16x16)
[vo/opengl] Using FBO format 0x805b.
[vo/opengl] No advanced processing required. Enabling dumb mode.
[vo/opengl] Texture for plane 0: 3840x2160
[vo/opengl] Texture for plane 1: 1920x1080
[vo/opengl] Texture for plane 2: 1920x1080
[cplayer] set video colors output-levels=0 
[statusline] V: 00:00:01 / 00:04:57 (0%) Dropped: 21
[cplayer] Run command: quit, flags=9, args=[0]
[cplayer] EOF code: 6  
[vd] Uninit video.
[cplayer] finished playback, success (reason 3)
[cplayer] 
[cplayer] 
[cplayer] Exiting... (Quit)
[osx] Exiting...
[ytdl_hook] Exiting...
[osc] Exiting...
[vo/opengl] flushing shader cache
```

### Sample files

* I used this file as example, but happens with other HLG files as well http://4kmedia.org/lg-cymatic-jazz-hdr-hlg-uhd-4k-demo/

**Comment by haasn** [comment]
I can't reproduce this on my end because starting one of these partial HEVC dumps midway always produces a corrupt gray image for me. How do you avoid that?

**Comment by kodabb** [comment]
@haasn yes, a couple of those sample files are actually bad hevc encodes, so I get gray images as you do. However the file I posted has some colored content from the start, and decodes correctly with a recent version of libavcodec even if you seek it

**Comment by kodabb** [comment]
Found the root problem: an HLG file may have its transfer characteristic in a separate SEI. I added support for that in ffmpeg but I just noticed that the very first frame does not get updated. mpv reads that as bt709/bt2020, sets up the appropriate gamma, then gets a new frame with arib-std-b67, and has to reset rendering with the proper new gamma (or tonemap).
I'll send a fix for libavcodec, sorry for the noise.
